### PR TITLE
Direct suggestions to ideas.storj.io

### PR DESCRIPTION
**Pull request by barlock**
What: 
Updating the readme to direction suggestions to ideas.storj.io

Why:
My normal use of GitHub issues is to use them to suggest changes as well as bug reports. Since it can be difficult to get to rocket chat (#1795), this seemed like a good way to help folks find where to go to contribute ideas.

Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

**Comment by barlock** [comment]
It's been signed 🎉

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ retrieve those files!
 
 All of our code for Storj v3 is open source. Have a code change you think would make Storj better? Please send a pull request along! Make sure to sign our [Contributor License Agreement (CLA)](https://docs.google.com/forms/d/e/1FAIpQLSdVzD5W8rx-J_jLaPuG31nbOzS8yhNIIu4yHvzonji6NeZ4ig/viewform) first. See our [license section](#license) for more details.
 
-Have comments, bug reports, or suggestions? Want to propose a PR before hand-crafting it? Jump on to our [Rocketchat](https://community.storj.io) and join the [#dev channel](https://community.storj.io/channel/dev) to say hi to the developer community and to talk to the Storj core team.
+Have comments, or bug reports? Want to propose a PR before hand-crafting it? Jump on to our [Rocketchat](https://community.storj.io) and join the [#dev channel](https://community.storj.io/channel/dev) to say hi to the developer community and to talk to the Storj core team. Want to vote on or suggest a new features? Post it on [ideas.storj.io](https://ideas.storj.io).
```
Comment (stefanbenten) [inline README.md]: Please remove the unnecessary comma 👍

Comment (stefanbenten) [reply README.md]: Also you could make the newly added sentence a new paragraph, to highlight it even better.

Comment (stefanbenten) [reply README.md]: feature instead of features or remove the _a_ to allow the plural

Comment (barlock) [reply README.md]: Done!

**Comment by barlock** [comment]
Huh, strange that CLA bot still didn't like it. Did I fill it out wrong?

**Comment by stefanbenten** [comment]
@barlock 
We have to add you to the CLA list, as this is not done automatically.

**Pull request merged**